vendor/ruby/astexport.rb

```diff
@@ -547,6 +553,12 @@ def visit_binary(node)
   { ast_type: type, left: visit(left), operator: op, right: visit(right) }
 end
 
+def visit_module(node)
+  # ["module", const_path_ref, bodystmt]
+  type, name, body = node
+  { ast_type: type, name: visit(name), body: visit(body), node: node }
```

Comment:
Please remove the extra node object it is not needed

Reply:
Apologies, that was a debugging artifact. I've removed it now :+1: